misc/ArmstrongNumber.c

```diff
@@ -0,0 +1,64 @@
+//A number is called as Armstrong number if sum of cubes of digits of number is equal to the number itself.
```

Comment:
the definition is not perfect. Please change it to something like

> Armstrong number is a number that is the sum of its own digits each raised to the power of the number of digits. 

[Narcissistic Number - wikipedia](https://en.wikipedia.org/wiki/Narcissistic_number)